src/nnfusion/engine/pass/codegen/cuda_codegen_pass.cpp

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
u may need to change code [here](https://github.com/lynex/nnfusion/blob/fe2b29e61950105b9d0f5cfd13870fd764f69980/src/nnfusion/engine/pass/codegen/cuda_codegen_pass.cpp#L421) from 
```
 sort(pairs.begin(),
             pairs.end(),
             [](std::pair<string, vector<nnfusion::ir::Instruction::Pointer>>& a,
                std::pair<string, vector<nnfusion::ir::Instruction::Pointer>>& b) {
                 if (a.first.find("default_") != string::npos)
                     return false;

                 if (b.first.find("default") != string::npos)
                     return false;

                 int pos_a = a.first.find("async_");
                 int pos_b = b.first.find("async_");
                 if (pos_a >= 0 && pos_b >= 0)
                 {
                     string delimiter("_");
                     std::string d1 = a.first.substr(a.first.find(delimiter) + 1);
                     d1 = d1.substr(0, d1.find(delimiter));
                     std::string d2 = b.first.substr(b.first.find(delimiter) + 1);
                     d2 = d2.substr(0, d2.find(delimiter));
                     return std::stoi(d1) < std::stoi(d2);
                 }
                 else
                 {
                     return a.first > b.first;
                 }
             });
```
into 
```
       sort(pairs.begin(),
             pairs.end(),
             [](std::pair<string, vector<nnfusion::ir::Instruction::Pointer>>& a,
                std::pair<string, vector<nnfusion::ir::Instruction::Pointer>>& b) {
                 int pos_a = a.first.find("async_");
                 int pos_b = b.first.find("async_");
                 if (pos_a >= 0 && pos_b >= 0)
                 {
                     string delimiter("_");
                     std::string d1 = a.first.substr(a.first.find(delimiter) + 1);
                     d1 = d1.substr(0, d1.find(delimiter));
                     std::string d2 = b.first.substr(b.first.find(delimiter) + 1);
                     d2 = d2.substr(0, d2.find(delimiter));
                     return std::stoi(d1) < std::stoi(d2);
                 }
                 else
                 {
                     return a.first > b.first;
                 }
             });
```